app/models/concerns/hmrc_interface_resultable.rb

```diff
@@ -38,6 +38,18 @@ def clients_income_from_self_employment
         self_assessment_summary_tax_return_total_incomes_by_year&.join_compact_blank("\n")
     end
 
+    # returns decimal or zero
```

Comment:
Question: I find this column heading/mapping a little confusing. Should it clients_income_from_employment - taxable_pay (i.e. taxable earnings that are not from employment?)

Reply:
the heading/mapping is just to match what is in the spreadsheet. The value being returned is taken from taxablePay because case workers have identified this a source of info for pay from "other" (possibly all, including employment?!) sources. They were particularly interested in pay from Pensions, for example.

see the tab in the spreadsheet named"Copy Pension/other income" for an example.


We already have a separate [clients_income_from_employment](https://github.com/ministryofjustice/laa-assure-hmrc-data/pull/344/files#diff-a272db2bdf367b7713b6d69f025ce50cdc08b62b22c86bb14c87603de83e9f46R15) that uses `gross_earnings_for_nics`

Reply:
from examples we have seen, for an employed person, `taxablePay" does seems to be from "all" sources rather than "other" sources. We could deduct income from employment for their convenience but i would have a conversation with Helen about this.

Reply:
have emailed Helen with questions below, (cc'd you):
1. Should we name this column "Client's income from **all** sources"?
OR
2. Should we [be more clever and] keep it as "Client's income from other sources" but deduct income from employment from the value? - i.e. "Client's income from other sources (excluding employment)"

Reply:
Case workers have confirmed that they
> ... want to be able to differentiate between client income from employment and client income from other sources.

Reply:
> Question: I find this column heading/mapping a little confusing. Should it clients_income_from_employment - taxable_pay (i.e. taxable earnings that are not from employment?)

yes realise I had the values the wrong way round in this comment!